lib/routes/transactions/pos_payment_item.dart

```diff
@@ -42,7 +47,14 @@ class PosPaymentItem extends StatelessWidget {
               mainAxisAlignment: MainAxisAlignment.start,
               mainAxisSize: MainAxisSize.min,
               children: [
-                _valueText(context),
+                Column(
```

Comment:
It is better to wrap this component in a stream builder of account and take the account from there rather than push the account model to all payment items as the currency changes independently from the payment items.

Reply:
@ademar111190 I agree with you that if that blinks it is not better.
I am OK to leave it as is.